### DISCUSSION TITLE
How to load tflite model into ocr_predictor

### DISCUSSION THREAD

**Discussion by SavvaVyatkin** [mindee/doctr#1259] [Q&A]
How do I load my tflite model into the ocr_predictor for doctr?

**Comment by felixT2K** [comment]
Hi @SavvaVyatkin 👋🏼 ,

you can't load a tflite model into the `ocr_predictor` after the export it's on your own whatever you want to do with the exported model :) 

Example js: https://github.com/mindee/doctr-tfjs-demo
